manifests/testing-framework/testing-framework.yaml

```diff
@@ -73,20 +73,23 @@ metadata:
   name: testing-framework
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
 metadata:
   name: testing-framework
 rules:
   - apiGroups: [""]
```

Comment:
We can remove the rule for secret manipulation, as the secrets needed for the credentials are already in the cluster.

Reply:
Speaking of, we'll have to open a PR for that too